fix travis

Travis is failing on #3011 and #3020 with this error:

```
$ make test
env/bin/honcho -e defaults.env,tests/test.env,tests/local.env run ./recreate-schema.sh
==============================================================================
Recreating public schema ... 
psql: FATAL:  database "travis" does not exist
make: *** [test-schema] Error 2
The command "make test" exited with 2.
Done. Your build exited with 1.
```

What's up? [IRC](https://botbot.me/freenode/gratipay/2014-12-17/?msg=27829353&page=2)

--- a/Makefile
+++ b/Makefile
@@ -8,8 +8,8 @@ venv := "./vendor/virtualenv-1.11.6.py"
 test_env_files := defaults.env,tests/test.env,tests/local.env
 pip := $(env_bin)/pip
 honcho := $(env_bin)/honcho
-honcho_run := $(honcho) -e defaults.env,local.env run
-py_test := $(honcho) -e $(test_env_files) run $(env_bin)/py.test
+honcho_run := $(honcho) run -e defaults.env,local.env
+py_test := $(honcho) run -e $(test_env_files) $(env_bin)/py.test
 
 env: requirements.txt requirements_tests.txt setup.py
 	$(python) $(venv) \
@@ -42,7 +42,7 @@ py: env
 	$(honcho_run) $(env_bin)/python -i -c 'from gratipay.wireup import env, db; db = db(env())'
 
 test-schema: env
-	$(honcho) -e $(test_env_files) run ./recreate-schema.sh
+	$(honcho) run -e $(test_env_files) ./recreate-schema.sh
 
 pyflakes: env
 	$(env_bin)/pyflakes bin gratipay tests

--- a/README.md
+++ b/README.md
@@ -159,7 +159,7 @@ If Gratipay launches successfully it will look like this:
 
 ```
 $ make run
-PATH=env/bin:{lots-more-of-your-own-PATH} env/bin/honcho -e defaults.env,local.env run web
+PATH=env/bin:{lots-more-of-your-own-PATH} env/bin/honcho run -e defaults.env,local.env web
 2014-07-22 14:53:09 [1258] [INFO] Starting gunicorn 18.0
 2014-07-22 14:53:09 [1258] [INFO] Listening at: http://0.0.0.0:8537 (1258)
 2014-07-22 14:53:09 [1258] [INFO] Using worker: sync
@@ -367,7 +367,7 @@ To invoke py.test directly you should use the `honcho` utility that comes
 with the install. First `make tests/env`, activate the virtualenv and then:
 
     [gratipay] $ cd tests/
-    [gratipay] $ honcho -e defaults.env,local.env run py.test
+    [gratipay] $ honcho run -e defaults.env,local.env py.test
 
 Local Database Setup
 --------------------

--- a/tests/js/utils/session.js
+++ b/tests/js/utils/session.js
@@ -2,7 +2,7 @@ var exec = require('sync-exec');
 
 module.exports = function createSession(user) {
     return exec(
-        './env/bin/honcho -e defaults.env,tests/test.env,tests/local.env ' +
-        'run ./env/bin/python ./tests/js/utils/auth-helper.py ' + user
+        './env/bin/honcho run -e defaults.env,tests/test.env,tests/local.env ' +
+        './env/bin/python ./tests/js/utils/auth-helper.py ' + user
     ).stdout.slice(0, -1);
 };

